src/main/scala/com/atomist/rug/kind/ServiceLoaderTypeRegistry.scala

```diff
@@ -6,7 +6,7 @@ import com.atomist.rug.RugRuntimeException
 import com.atomist.rug.spi._
 import com.typesafe.scalalogging.LazyLogging
 
-import scala.collection.JavaConverters._
+import _root_.scala.collection.JavaConverters._
```

Comment:
This is a special Scala syntax for handling package conflicts. There's a `scala` package underneath `kind` now. Importing anything starting with `scala` will be treated as a relative import.